common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -250,26 +247,34 @@
             "error"
         ],
         "@typescript-eslint/no-unsafe-assignment": [
-            "error"
+            "warn"
         ],
         "@typescript-eslint/no-unsafe-call": [
-            "error"
+            "warn"
         ],
         "@typescript-eslint/no-unsafe-member-access": [
-            "error"
+            "warn"
         ],
         "@typescript-eslint/no-unsafe-return": [
             "error"
         ],
         "@typescript-eslint/no-unused-expressions": [
-            "error"
+            "warn"
```

Comment:
I think we should keep errors on no-unused-expressions

Reply:
I can't remember why I changed this, so I'll change it back, remember why I lowered it 😆 then RAGING DEBATE can ensue.